ArkEcosystem/explorer
# refactor: missing multisignature tooltip

## Checklist

https://app.clickup.com/t/m9but8

When viewing the amount of the multipayment wallet transactions, it filters the amount sent to itself and shows a tooltip with the amount.

Plus: updates the amount boxes padding to match the design, and now it shows the box on mobile (used to show only the amount)

### To test

1. Open a wallet (for example `/wallets/DArvWfH5nMDT38tWmo5k461vMQpRXHQWX9` and filter for multipayment transactions
2. See the tooltips
3. Check mobile, compact, and expanded tables, and dark mode

![image](https://user-images.githubusercontent.com/17262776/121722988-9ac8c780-caab-11eb-9912-6737805ee35d.png)
![image](https://user-images.githubusercontent.com/17262776/121723007-a0bea880-caab-11eb-91c2-2507c27b7e8e.png)
![image](https://user-images.githubusercontent.com/17262776/121723039-a9af7a00-caab-11eb-9969-e1ef0eb6ca85.png)

![image](https://user-images.githubusercontent.com/17262776/121722955-92708c80-caab-11eb-9d92-9ebd4d6c3c59.png)
